README.md

```diff
@@ -29,7 +31,7 @@ Instructions about how to build and install the Solidity compiler can be found i
 A "Hello World" program in Solidity is of even less use than in other languages, but still:
 
 ```
-pragma solidity ^0.4.16;
+pragma solidity ^0.5.2;
 
 contract HelloWorld {
   function helloWorld() external pure returns (string memory) {
```

Comment:
10 lines down there are links to the documentation with hardcoded version number - we should use something like `latest` or `develop`.

Reply:
Are you saying to add a variable into the RST file and update everything at once? Or are you recommending a change to the `pragma` directive?

Reply:
No, I'm saying the link could be https://solidity.readthedocs.io/en/latest/solidity-by-example.html#voting

Reply:
(down _from_ here)